pages/realtime.vue

```diff
@@ -432,8 +432,8 @@ export default {
       this.sse.close();
     }
   },
-  updated: function() {
-    this.$nextTick(function() {
+  updated: ()=> {
```

Comment:
Don't use arrow functions on an options property

Reply:
Okay, I will revert this back . I'm really sorry for the Trouble Caused